README.md

```diff
@@ -171,6 +171,14 @@ const paginateConfig: PaginateConfig<CatEntity> {
    */
   searchableColumns: ['name', 'color'],
 
+  /**
+   * Required: false
+   * Type: (keyof CatEntity)[]
+   * Description: This columns will be selected only.
+   * You could also include relationships colums, like toy.name
+   */
+  selectableColumns: ['name', 'color'],
```

Comment:
I would prefer to directly use the typeorm naming `select` for this prop - similar as we do with `where`.

`selectableColumns` implies that something is optionally "selectable". I think this would fit better for custom field selection for the query params like: what fields are selectable [A, B, C] ? and then `/cats?select=B,C`

Reply:
Make sense. I honestly was not very comfortable with the name but I did not come up with anything else 😅